docker/indexer/stages/preparation/preparation.go

```diff
@@ -177,16 +177,8 @@ func (s *Stage) processGit(ctx context.Context, repoCfg *config.RepoConfig) erro
 			version   string
 			commitTag = ref.Name().String()
 		)
-		if repoCfg.VersionRE != "" {
-			versRE, err := regexp.Compile(repoCfg.VersionRE)
-			if err != nil {
-				return err
-			}
-			version = versRE.FindString(commitTag)
-		}
-		if version == "" {
-			version = genericVersionRE.FindString(commitTag)
-		}
+		verComponents := genericVersionRE.FindAllString(commitTag, -1)
+		version = strings.Join(verComponents, "-")
```

Comment:
Should we just use "." as the separator here since this version is what will be returned in our responses? "." probably makes a lot more sense to API users than "-".